src/ProjectNeon/Assets/Data/Templates/Member.cs

```diff
@@ -7,6 +7,7 @@ public class Member : Target
     [SerializeField] private Character character;
     [SerializeField] private Deck deck;
     [SerializeField] public int hp;
+    [SerializeField] private TurnAI player = new NoAI();

```

Comment:
This is the wrong design direction. 

We care about `Member` only so far as they can be targeted in battle, and things can happen to their mutate state, such as they can lose HP, gain temporary Armor, receive Buffs, receive Debuffs...etc.

Not all `Members` will have AIs or Players. Members probably should also not expose their `Deck`.